/*const handleError = (res, func) => {
    try {
        func();
    } catch (error) {
        res.status(500).json({ error: error.toString() });
    }
};

module.exports = {
    handleError,
};*/
